src/components/modal_pages/SignInStep2.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+
+  <!-- ToDo: Step2 - Download -->
+  <section class="docdog-modal">
+    <header class="docdog-modal__head">
```

Comment:
Header part here is shared for all Modal pages right ?

Reply:
Yes, it isn't need a new logic.